package/python-kiwi-spec-template

```diff
@@ -125,12 +125,10 @@ Provides:       kiwi-image:tbz
 # tools conditionally used by kiwi
 %if 0%{?fedora} || 0%{?rhel} >= 8
 Recommends:     gnupg2
-Recommends:     debootstrap
```

Comment:
This needs to change to `Recommends: apt`.

Reply:
done